### DISCUSSION TITLE
Ubuntu last LTS version

### DISCUSSION THREAD

**Discussion by KevinMalot** [cybertec-postgresql/pgwatch2#482] [Q&A]
Hello

Is there any need to keep an old version of Ubuntu base image or should we consider updating the Dockerfiles to use a more recent LTS one ? 
Such as 20.04 or 22.04

Regards,

Kevin

**Comment by ghost** [comment]
I don't think that it necessary to keep old versions of Ubuntu base image...
I have ever try to update some of Them. Example : webui. (I need it due to my infrastructure deploy mode)
